content/bn/virtualization.md

```diff
@@ -5,27 +5,28 @@ category: প্রযুক্তি
 tags: ["fundamental", "infrastructure", "methodology"]
 ---
 
-## এটা কি
+## এটা কি ?
 
 ভার্চুয়ালাইজেশন, ক্লাউড নেটিভ কম্পিউটিং প্রসঙ্গে,
 একটি শারীরিক কম্পিউটার নেওয়ার প্রক্রিয়াকে বোঝায়, কখনও কখনও একটি সার্ভার বলা হয়,
 এবং এটি একাধিক বিচ্ছিন্ন অপারেটিং সিস্টেম চালানোর অনুমতি দেয়।
 সেই বিচ্ছিন্ন অপারেটিং সিস্টেম এবং তাদের ডেডিকেটেড কম্পিউট রিসোর্স (সিপিইউ, মেমরি এবং নেটওয়ার্ক)
-ভার্চুয়াল মেশিন বা ভিএম হিসাবে উল্লেখ করা হয়।
+ভার্চুয়াল মেশিন বা ভিএম (VM) হিসাবে উল্লেখ করা হয়।
 যখন আমরা একটি [ভার্চুয়াল মেশিন](/bn/virtual-machine/) সম্পর্কে কথা বলি, তখন আমরা একটি সফ্টওয়্যার-সংজ্ঞায়িত কম্পিউটারের কথা বলছি।
-এমন কিছু যা দেখতে এবং একটি বাস্তব কম্পিউটারের মতো কাজ করে কিন্তু অন্যান্য ভার্চুয়াল মেশিনের সাথে হার্ডওয়্যার ভাগ করছে৷
+এমন কিছু দেখতে যা একটি বাস্তব কম্পিউটারের মতো কাজ করে, কিন্তু অন্যান্য ভার্চুয়াল মেশিনের সাথে হার্ডওয়্যার ভাগ করে৷
 [ক্লাউড কম্পিউটিং](/bn/cloud-computing/) প্রাথমিকভাবে ভার্চুয়ালাইজেশন প্রযুক্তি দ্বারা চালিত।
 উদাহরণ হিসেবে, আপনি AWS থেকে একটি "কম্পিউটার" লিজ দিতে পারেন - সেই কম্পিউটারটি আসলে একটি VM।
 
 ## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
 
-ভার্চুয়ালাইজেশন শারীরিক হার্ডওয়্যার ব্যবহারের উন্নতি সহ বেশ কয়েকটি সমস্যার সমাধান করে
-একই ফিজিক্যাল মেশিনে আরও অ্যাপ চালানোর অনুমতি দিয়ে
-নিরাপত্তার জন্য একে অপরের থেকে বিচ্ছিন্ন থাকা অবস্থায়।
+ভার্চুয়ালাইজেশন নিরাপত্তার জন্য একে অপরের থেকে বিচ্ছিন্ন থেকে একই অবস্থায় ফিজিক্যাল মেশিনে
+আরও অ্যাপ চালানোর অনুমতি দেয় এবং 
+শারীরিক হার্ডওয়্যার ব্যবহারের উন্নতি সহ বেশ কয়েকটি সমস্যার সমাধান করে।
```

Comment:
```suggestion
ফিজিক্যাল হার্ডওয়্যার ব্যবহারের উন্নতি সহ বেশ কয়েকটি সমস্যার সমাধান করে।
```

Reply:
I have used the term "শারীরিক" it in order to maintain the proper translation of the term "Virtualization" in Bengali.